### PR TITLE
Fix output and by default create role to make the module non breaking

### PR DESCRIPTION
**Pull request by niroz89**

### CODE REVIEW

**Review by lawliet89 — APPROVED** [review]
Update README

File: variables.tf [hunk 1]
```diff
@@ -190,7 +190,7 @@ variable "external_secrets_secrets_manager_arns" {
 variable "create_default_irsa" {
   description = "Create default IRSA for service account"
   type        = bool
-  default     = false
+  default     = true
```
Comment (uchinda-sph) [inline variables.tf:193]: why we are creating irsa.? we have move that to parent module right, with s3 permissions

Comment (niroz89) [reply variables.tf:193]: the module was creating irsa earlier, so can release it as a non breaking version, when I'm calling, I will set it explicitly false to not create irsa.

**Review by uchinda-sph — APPROVED** [review]
LGTM

**Pull request merged**